src/vs/workbench/contrib/preferences/browser/settingsTreeModels.ts

```diff
@@ -570,9 +570,8 @@ function trimCategoryForGroup(category: string, groupId: string): string {
 }
 
 export function isExcludeSetting(setting: ISetting): boolean {
-	return setting.key === 'files.exclude' ||
-		setting.key === 'search.exclude' ||
-		setting.key === 'files.watcherExclude';
+	return setting.key in
+		['files.exclude', 'search.exclude', 'files.watcherExclude', 'explorer.autoRevealExclude'];
```

Comment:
This does not do what it looks like in JS (see the [`in` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)): `'hey' in ['hey', 'foobar'] === false`

If necessary to change to checking from an array like this (which is contrary to rest of file), should use `['...'].includes(...)`.

Reply:
You're right,  I was thinking of python syntax when I wrote this.  Need reverting

Reply:
Fixed